diff --git a/build/features.gypi b/build/features.gypi
index 08ea11a..3c952ff 100644
--- a/build/features.gypi
+++ b/build/features.gypi
@@ -54,7 +54,7 @@
 
     # Enable ECMAScript Internationalization API. Enabling this feature will
     # add a dependency on the ICU library.
-    'v8_enable_i18n_support%': 1,
+    'v8_enable_i18n_support%': 0,
 
     # Enable compiler warnings when using V8_DEPRECATED apis.
     'v8_deprecation_warnings%': 0,
diff --git a/build/standalone.gypi b/build/standalone.gypi
index 3d4f804..2dde3fc 100644
--- a/build/standalone.gypi
+++ b/build/standalone.gypi
@@ -36,7 +36,7 @@
     'clang%': 0,
     'visibility%': 'hidden',
     'v8_enable_backtrace%': 0,
-    'v8_enable_i18n_support%': 1,
+    'v8_enable_i18n_support%': 0,
     'msvs_multi_core_compile%': '1',
     'mac_deployment_target%': '10.5',
     'variables': {
@@ -180,7 +180,6 @@
         'conditions': [
           ['component=="static_library"', {
             'defines': [
-              '_HAS_EXCEPTIONS=0',
             ],
           }],
         ],
@@ -203,7 +202,7 @@
               ['component=="shared_library"', {
                 'ExceptionHandling': '1',  # /EHsc
               }, {
-                'ExceptionHandling': '0',
+                'ExceptionHandling': '1',
               }],
             ],
           },
diff --git a/build/toolchain.gypi b/build/toolchain.gypi
index c1066eb..8ee5d76 100644
--- a/build/toolchain.gypi
+++ b/build/toolchain.gypi
@@ -460,7 +460,7 @@
                   ['component=="shared_library"', {
                     'RuntimeLibrary': '3',  # /MDd
                   }, {
-                    'RuntimeLibrary': '1',  # /MTd
+                    'RuntimeLibrary': '3',  # /MDd
                   }],
                 ],
               }],
@@ -475,7 +475,7 @@
                   ['component=="shared_library"', {
                     'RuntimeLibrary': '3',  # /MDd
                   }, {
-                    'RuntimeLibrary': '1',  # /MTd
+                    'RuntimeLibrary': '3',  # /MDd
                   }],
                 ],
               }],
@@ -490,7 +490,7 @@
                   ['component=="shared_library"', {
                     'RuntimeLibrary': '3',  #/MDd
                   }, {
-                    'RuntimeLibrary': '1',  #/MTd
+                    'RuntimeLibrary': '3',  #/MDd
                   }],
                   ['v8_target_arch=="x64"', {
                     # TODO(2207): remove this option once the bug is fixed.
@@ -503,10 +503,10 @@
           'VCLinkerTool': {
             'conditions': [
               ['v8_optimized_debug==0', {
-                'LinkIncremental': '2',
+                'LinkIncremental': '1',
               }],
               ['v8_optimized_debug==1', {
-                'LinkIncremental': '2',
+                'LinkIncremental': '1',
               }],
               ['v8_optimized_debug==2', {
                 'LinkIncremental': '1',
